Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Delete.ts

```diff
@@ -29,15 +30,14 @@ export async function deleteDocumentByDocumentUuid(
   try {
     await client.query('BEGIN');
```

Comment:
Suggest pushing this down to `SqlHelper` also. I don't think Delete/Read/Update/Upsert.ts should ever call `client.query()`. Leave that to the `SqlHelper` layer.